.github/actions/issue-assigner/action.yml

```diff
@@ -0,0 +1,34 @@
+name: assign-k6-maintainer
+description: Assign a k6 maintainer to an issue
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/github-script@v6
+      with:
+        script: |
+          const assignees = ['mstoykov', 'codebien', 'olegbespalov', 'oleiade'];
```

Comment:
Maybe let's first test that it works across repos and then we can start changing it?

But yeah - it will probably be nice if this reads the CodeOwners file ?

Reply:
Generally, yeah it could have many more features, depending on our capacity and desire to maintain it, but as the first iteration, even the current version should cover our main pain, the issues discovery in other repos.

Reply:
For the record, I've changed my mind, and we shouldn't provide input here but use a static list to keep this as the single source of truth and have no way to create the case where the list will be distributed and overrided.

Reply:
Having this list fetched directly from the GitHub team's members would be great. Then we could also inspect the members' status and exclude the busy people (but we require https://github.com/orgs/community/discussions/42788).
Ofc, all of this after we have validated the current implementation, that sounds perfect as a starting point.